File: Tech Module/Software Technologies/JAVA/Calculator WEB/src/main/java/com/softuni/entity/Calculator.java
package com.softuni.entity;

public class Calculator {

        private double leftOperand;
        private double rightOperand;
        private String operator;

    public Calculator(double leftOperand, double rightOperand, String operator) {
        this.leftOperand = leftOperand;
        this.rightOperand = rightOperand;
        this.operator = operator;
    }

    public double getLeftOperand() {
        return leftOperand;
    }

    public void setLeftOperand(double leftOperand) {
        this.leftOperand = leftOperand;
    }

    public double getRightOperand() {
        return rightOperand;
    }

    public void setRightOperand(double rightOperand) {
        this.rightOperand = rightOperand;
    }

    public String getOperator() {
        return operator;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public double calculateResult()
    {
        double leftOperand = this.getLeftOperand();
        double rightOperand = this.getRightOperand();
        String operator = this.getOperator();
        double result = 0.0;

        if (operator.equals("+"))
        {
            result = leftOperand + rightOperand;
        }
        else if (operator.equals("-"))
        {
            result = leftOperand - rightOperand;
        }
        else if (operator.equals("*"))
        {
            result = leftOperand * rightOperand;
        }
        else if (operator.equals("/"))
        {
            result = leftOperand / rightOperand;
        }
        else if (operator.equals("^"))
        {
            result = Math.pow(leftOperand,rightOperand);
        }
        else if (operator.equals("%"))
        {
            result = leftOperand % rightOperand;
        }

        return result;
    }
}

File: Tech Module/Software Technologies/PHP/Calculator/src/CalculatorBundle/Entity/Calculator.php
<?php

namespace CalculatorBundle\Entity;

class Calculator
{
    /**
     * @var float
     */

    private $leftOperand;

    /**
     * @var float
     */
    private $rightOperand;

    /**
     * @var string
     */

    private $operator;

    /**
     * @return float
     */
    public function getLeftOperand(): ?float
    {
        return $this->leftOperand;
    }

    /**
     * @param float $leftOperand
     */
    public function setLeftOperand(float $leftOperand)
    {
        $this->leftOperand = $leftOperand;
    }

    /**
     * @return float
     */
    public function getRightOperand(): ?float
    {
        return $this->rightOperand;
    }

    /**
     * @param float $rightOperand
     */
    public function setRightOperand(float $rightOperand)
    {
        $this->rightOperand = $rightOperand;
    }

    /**
     * @return string
     */
    public function getOperator(): ?string
    {
        return $this->operator;
    }

    /**
     * @param string $operator
     */
    public function setOperator(string $operator)
    {
        $this->operator = $operator;
    }

    /**
     * @return float
     */
    public function calculateResult(): ?float
    {


        $result = 0;

        $operator = $this->getOperator();
        $leftOperand = $this->getLeftOperand();
        $rightOperand = $this->getRightOperand();

        switch ($operator) {
            case '+':

                $result = $leftOperand + $rightOperand;
                break;
            case '-':
                $result = $leftOperand - $rightOperand;

                break;
            case '*':
                $result = $leftOperand * $rightOperand;

                break;
            case '/':
                $result = $leftOperand / $rightOperand;

                break;
            case '%':
                $result = $leftOperand % $rightOperand;

                break;
            case '^':
                $result = pow($leftOperand, $rightOperand);

                break;
        }

        return $result;


    }


}